docs/devel/reporting.rst

```diff
@@ -1,14 +1,14 @@
 Translation progress reporting
 ==============================
 
-Reporting features give insight into how a translation progresses over a given
-period. A summary of contributions to any given component over time is
-provided. The reporting tool is found in the :guilabel:`Insights` menu of any
-translation component, project or on the dashboard:
+Reporting features give insight into translation progress over a given period.
+A summary of contributions to any given component over time is provided.
+The reporting tool is found in the :guilabel:`Insights` menu of any
+translation component and project, and shown on the dashboard:
```

Comment:
```suggestion
translation component and project, and possibly shown on the dashboard:
```

Reply:
not needed as it changes the meaning of the description.

Reply:
True. Seems it got fixed properly in the interim.